classes/module/Module.php

```diff
@@ -3297,10 +3335,42 @@ public function removeOverride($classname)
             }
 
             $to_delete = preg_match('/<\?(?:php)?\s+(?:abstract|interface)?\s*?class\s+' . $classname . '\s+extends\s+' . $classname . 'Core\s*?[{]\s*?[}]/ism', $code);
+
+            if (!$to_delete) {
+                // To detect if the class has remaining code, we dynamically create a class which contains the remaining code.
+                eval(
+                    preg_replace(
+                        [
+                            '#^\s*<\?(?:php)?#',
+                            '#class\s+' . $classname . '\s+extends\s+([a-z0-9_]+)(\s+implements\s+([a-z0-9_]+))?#i',
+                        ],
+                        [
+                            ' ',
+                            'class ' . $classname . 'OverrideOriginal_check' . $uniq . ' extends \stdClass',
+                        ],
+                        $code
+                    )
+                );
+
+                // Then we use ReflectionClass to analyze what this code actually contains
+                $override_class = new ReflectionClass($classname . 'OverrideOriginal_check' . $uniq);
```

Comment:
I'm a little bit afraid of performance if we install many modules in a short amount of time... like the install process 🤔 

Would like @PierreRambaud opinion on this one

Reply:
This one is executed only during the removal, so if we have no other choice than using the reflection that's ok